
### Resources(Adapt from the following)
https://www.bioconductor.org/help/course-materials/2015/BioC2015/methylation450k.html

https://github.com/STAT540-UBC/STAT540-UBC.github.io/blob/master/seminars/seminars_winter_2020/seminar7/sm07_methylation.md  
 
Need to check licensing but these are places to start

### Datasets:
Built in from minfi, Something from GEO (probably the same one 540 uses)

### Tasks we want students to learn:
Load in raw rgset data and convert to methylset and beta vals
Perform basic QC


Simple analysis (differential methylation)
Should we add gene set enrichment or clustering or is that irrelevant?
Clustering I think would make sense
Heat map to see upregulated and downregulated (?)
Maybe mapping the CpGs back to which regions of the genome they are in and then if
They are in genes performing enrichment analysis (?)


Questions we can ask students:

